src/snowflake/snowpark/_internal/utils.py

```diff
@@ -182,6 +182,9 @@
 
 SUPPORTED_TABLE_TYPES = ["temp", "temporary", "transient"]
 
+PIVOT_VALUES_NONE_OR_DATAFRAME_WARNING = "Parameter values is Optional or DataFrame is in private preview since v1.15.0.  Do not use it in production."
```

Comment:
can you add the word, "pivot" here somewhere? The warning has no context on the operation that we are trying to perform.

Reply:
Done.